src/lib/dayjs.js

```diff
@@ -3,4 +3,6 @@ import isBetween from 'dayjs/plugin/isBetween'
 
 dayjs.extend(isBetween)
 
+export const DATE_FORMAT = 'D MMM YY, h:mm A'
```

Comment:
i think that we should add it [here](https://github.com/aragon/court-dashboard/blob/77179b3a7a730728e41ae95b6701e2dd995665f8/src/utils/date-utils.js#L3)